burf-theme/content/_galleries.scss

```diff
@@ -78,17 +78,17 @@ $margin-gallery: 													10px !default;
 // Since: 1.4.0
 
 .gallery-item {
-	border-right: $margin-gallery solid transparent;
+	border-right: var(--gallery-margin, #{$margin-gallery}) solid transparent;
```

Comment:
margin for the border? I see the foundation variables did that though.

Reply:
ah I see the gallery uses transparent borders for the layout? That seems like something we should change in the new theme framework.